invenio_administration/assets/semantic-ui/js/invenio_administration/src/formik/AdminForm.js

```diff
@@ -15,13 +15,17 @@ import mapValues from "lodash/mapValues";
 export class AdminForm extends Component {
   constructor(props) {
     super(props);
-    const { resource, formFields } = props;
+    const { resource, resourceSchema: resourceSchema } = props;
 
     this.state = {
       error: undefined,
       formData: resource
         ? resource
-        : mapValues(formFields, function (value) {
+        : mapValues(resourceSchema, function (value) {
```

Comment:
I find it interesting that we were returning always `""`? What is the need of this `func`?

Reply:
To create fields with empty values. This form renders for `create` and `edit` pages. In case of `edit` the fields are prefilled with `resource` (existing banner data), but in case of `create` page fields have to be empty.